### PR TITLE
lbc: trim CPU requests/limits on unit-test

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: config/jobs/kubernetes-sigs/aws-load-balancer-controller/aws-alb-ingress-controller-presubmits.yaml [hunk 1]
```diff
@@ -42,10 +42,10 @@ presubmits:
         - unit-test
         resources:
           limits:
-            cpu: 8
+            cpu: 6
             memory: 4Gi
           requests:
-            cpu: 8
+            cpu: 6
```
Comment (kishorj) [inline config/jobs/kubernetes-sigs/aws-load-balancer-controller/aws-alb-ingress-controller-presubmits.yaml:48]: Do we need the request? The pod scheduler could fail if it is unable to find the request. We can remove the CPU request, and rely on the limit.

Comment (johngmyers) [reply config/jobs/kubernetes-sigs/aws-load-balancer-controller/aws-alb-ingress-controller-presubmits.yaml:48]: Request must match limit or the PR is rejected.

Comment (kishorj) [reply config/jobs/kubernetes-sigs/aws-load-balancer-controller/aws-alb-ingress-controller-presubmits.yaml:48]: I feel limit of 6 is still high, lets set it to 2, and we modify the UTs instead to decrease the resource usage.

Comment (johngmyers) [reply config/jobs/kubernetes-sigs/aws-load-balancer-controller/aws-alb-ingress-controller-presubmits.yaml:48]: Let's set it high to unblock things, then once it's working we can experiment with non-blocking presubmits to find the real value.

**Review by kishorj — APPROVED** [review]

**Pull request merged**